src/generic/Misc/Rings.jl

```diff
@@ -28,6 +28,29 @@ function root(a::RingElem, n::Int)
     return b
 end
 
+@doc raw"""
+    falling_factorial(x::RingElement, n::Integer)
+
+Return the falling factorial of $x$, i.e. $x(x - 1)(x - 2)\cdots (x - n + 1)$.
+If $n < 0$ we throw a `DomainError()`.
+
+# Examples
+
+```jldoctest
+julia> R, x = ZZ[:x];
+
+julia> falling_factorial(x, 1)
+x
+
+julia> falling_factorial(x, 2)
+x^2 - x
+
+julia> falling_factorial(4, 2)
+12
+```
+"""
+falling_factorial(x::RingElement, n::Integer) = (-1)^n*rising_factorial(-x, n)
```

Comment:
This is not reported as covered -- I think because we don't execute the doctest, because the function is not actually included in the documentation?

Perhaps we can change that and include both this function and `rising_factorial` into the manual. E.g. add a new section `Miscellaneous` to the end of `docs/src/ring.md` and include them there.